prefect-worker/repository.yaml

```diff
@@ -0,0 +1,10 @@
+name: prefect-worker
+description: prefect-worker deployed on plural
+category: DATA
+private: true
+icon: plural/icons/prefect-worker.png
+homepage: https://www.prefect.io/
+gitUrl: https://github.com/PrefectHQ/prefect
+notes: plural/notes.tpl
+tags:
+- tag: data
```

Comment:
I think this is documented in the README.md but you can add your user as a `contributor` here and that can keep the repo private but still allow you to install it for testing.